src/lunabot_firmware/excavator_door/xbox_toggle_bridge.py

```
@@ -0,0 +1,68 @@
+        description="Send a toggle command to the STM32 when Xbox B is pressed."
+    )
+    parser.add_argument("--port", required=True, help="Serial port, for example COM3")
+    parser.add_argument("--baud", type=int, default=9600, help="Serial baud rate")
+    args = parser.parse_args()
+
+    pygame.init()
+
+    try:
+        joystick = open_first_controller()
+        ser = serial.Serial(args.port, args.baud, timeout=0.1)
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)